connector/src/main/scala/com/basho/riak/spark/rdd/RiakTSRDD.scala

```diff
@@ -48,48 +49,32 @@ class RiakTSRDD[R] private[spark](
   extends RDD[R](sc, Seq.empty) with Logging {
 
   override def getPartitions: Array[Partition] = {
-     tsRangeFieldName match {
-       case None => SinglePartitionRiakTSPartitioner.partitions(connector.hosts, bucketName, schema, columnNames, query, whereConstraints, filters)
-       case Some(name) => RangedRiakTSPartitioner.partitions(connector.hosts, bucketName, schema, columnNames, filters, name, readConf)
-     } 
-  }
-
-  private def validateSchema(schema: StructType, columns: Seq[ColumnDescription]): Unit = {
-    val columnNames = columns.map(_.getName)
-
-    schema.fieldNames.diff(columnNames).toList match {
-      case Nil => columnNames.diff(schema.fieldNames) match {
-        case Nil =>
-        case diff =>
-          throw new IllegalArgumentException(s"Provided schema has nothing about the following fields returned by query: ${diff.mkString(", ")}")
+    if (filters.isEmpty)
+      SinglePartitionRiakTSPartitioner.partitions(connector.hosts, bucketName, schema, columnNames, query, whereConstraints)
```

Comment:
Please add a short comment about the situation when this partitioner will be used